pkg/terminal/terminal.go

```diff
@@ -210,22 +211,36 @@ func (t *Term) Run() (int, error) {
 	signal.Notify(ch, syscall.SIGINT)
 	go t.sigintGuard(ch, multiClient)
 
+	fns := trie.New()
+	cmds := trie.New()
+	funcs, _ := t.client.ListFunctions("")
+	for _, fn := range funcs {
+		fns.Add(fn, nil)
+	}
+	for _, cmd := range t.cmds.cmds {
+		for _, alias := range cmd.aliases {
+			cmds.Add(alias, nil)
+		}
+	}
+
 	t.line.SetCompleter(func(line string) (c []string) {
+		prefix := ""
 		if strings.HasPrefix(line, "break ") || strings.HasPrefix(line, "b ") {
+			prefix = "break "
+		}
+		if strings.HasPrefix(line, "continue ") || strings.HasPrefix(line, "c ") {
+			prefix = "continue "
+		}
+		if prefix != "" {
 			filter := line[strings.Index(line, " ")+1:]
```

Comment:
IMHO we should make this work better:

```
if spc := strings.LastIndex(line, " "); spc > 0 {
    prefix := line[:spc]
    funcs := fns.FuzzySearch(line[spc+1:])
    for _, f := range funcs {
         c = append(c, prefix+f)
    }
}

```

Reply:
This way it will also work with `break bpname fmt.Pri<TAB>`, now it doesn't.

Reply:
Done.